definitions/variable/macro-economy/tag_econ_sectors.yaml

```diff
@@ -0,0 +1,10 @@
+- Econ sectors:
+    - Agriculture:
+        description: the aggregated agricultural sectors (ISIC Rev.4 A)
+        tier: 1
+    - Industry:
+        description: the aggregated industrial sector (ISIC Rev.4 B-F)
+        tier: 1
+    - Services:
```

Comment:
This would be a change compared to the NAVIGATE variables, which used "Commercial" - are you sure that this change is worth the inconvenience to the teams that have to update their reporting?